.github/workflows/build-and-test.yml

```diff
@@ -572,6 +572,9 @@ jobs:
     needs: [publish-stable]
     if: startsWith(github.ref, 'refs/tags/v') && github.repository == 'open-telemetry/opentelemetry-collector-contrib'
     steps:
+      - name: Set Release Tag
+        id: github_tag
+        run: ./.github/workflows/scripts/set_release_tag.sh
```

Comment:
Instead of this, you can use `github.ref_name` as is done in the core repo: https://github.com/open-telemetry/opentelemetry-collector/blob/3b62976f35387b150244390e8f8286a820966963/.github/workflows/builder-release.yaml#LL31C20-L52C14

Reply:
done